### ISSUE TITLE
Issue: AI Sensy Integration for Whatsapp Channel provider

### ISSUE THREAD

**Issue by Lekhanrao** [OpenNyAI/Jugalbandi-Manager#180]
### Ticket Contents

To implement and integrate AI Sensy with Jugalbandi Manager

### Goals

Currently we have integrated one of the BSPs of Whatsapp Business API called ‘Pinnacle’. To extend the support with AI Sensy integration along with Pinnacle. 

### Expected Outcome

The stack should have capabilities to connect to Whatsapp Business API via AI Sensy

### Acceptance Criteria

1. All the capabilities implemented for Pinnacle need to be implemented for AI Sensy
2. Appropriate test cases should also be written for the same

### Implementation Details

The current implementation supports connecting with Whatsapp Business API via Pinnacle. All the functionalities implemented for Pinnacle to be developed for AI Sensy by referring to its official documentation.
Below are the type of messages that need to be handled -

- Text
- Audio
- Interactive

Also it refers the base channel handler - jb-lib/lib/channel_handler/channel_handler.py
For reference to Pinnacle implementation - jb-lib/lib/channel_handler/pinnacle_whatsapp_handler.py
For reference to writing tests - jb-lib/tests/channel_handler/test_whatsapp.py

### Mockups/Wireframes

Not Applicable

### Product Name

Jugalbandi

### Organisation Name

OpenNyAI

### Domain

Legal

### Tech Skills Needed

Python, Other

### Mentor(s)

@Lekhanrao @KaranrajM 

### Complexity

Medium

### Category

Backend

**Comment by RishiikeshBalaji** [comment]
hi. I would like to work on this issue? @Lekhanrao 
I have experience in building AI based use cases.

**Comment by Lekhanrao** [comment]
@RishiikeshBalaji, assigned ticket to you.

**Comment by RishiikeshBalaji** [comment]
thanks @Lekhanrao 
can we connect in discord to discuss further

**Comment by Lekhanrao** [comment]
Sure... we can... you can also post your questions here in GitHub. Karan is our Tech lead he can help answer your questions.

**Comment by RishiikeshBalaji** [comment]
@Lekhanrao I am currently setting up the dev environment. I have some doubts in setting up the env, could you help me through this if possible over a call?

After setting up the environment, I would like to understand the codebase properly and start my work

**Comment by Lekhanrao** [comment]
@RishiikeshBalaji, could you please list your doubts here related to setting up the env? @KaranrajM, can you help respond?

**Comment by RishiikeshBalaji** [comment]
<img width="827" alt="image" src="https://github.com/user-attachments/assets/81bde740-ba50-4e13-84d4-dc88517a2769">
I am following the documentation and trying to set up the env. from where can I get these api keys for .env-dev file?

**Comment by RishiikeshBalaji** [comment]
sure @KaranrajM will look into it. 
can we connect in discord any time when you are free to get a deep understanding of the problem statement?

**Comment by Lekhanrao** [comment]
Karan will be reaching out to you on Discord by today evening or early
tomorrow morning to explain.

Regards,
Lekha
Cell: 9008533001
Website: Agami <https://agami.in/>

**Comment by Lekhanrao** [comment]
@KaranrajM to connect with Rishikesh Balaji today. 
@RishiikeshBalaji , fyi

**Comment by Lekhanrao** [comment]
@RishiikeshBalaji , I understand Karan has connected with you and clarified your doubts. Pls let us know if there is an update

**Comment by Lekhanrao** [comment]
@KaranrajM and @RishiikeshBalaji is there any update on this please?

**Comment by RishiikeshBalaji** [comment]
@Lekhanrao yes, currently I am working on the requirement. I will update the progress as soon as possible

**Comment by RishiikeshBalaji** [comment]
@KaranrajM  I have updated the status in the discord chat, we need to discuss about that.

**Comment by Lekhanrao** [comment]
@KaranrajM , where you able to check with RishikeshBalaji?

**Comment by Lekhanrao** [comment]
@KaranrajM any update on this? were you able to check with Rishikesh balaji?

**Comment by Khan-Ramsha** [comment]
Hello, Would like to work on this issue, is this still active and available for contribution?

**Comment by keshavv-dadhich** [comment]
hi. I would like to work on this issue? @Lekhanrao 
I have experience in building AI based use cases.
Is this still active and available for contribution
